apps/docs/docusaurus.config.js

```diff
@@ -72,6 +72,36 @@ module.exports = {
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
+      metadata: [
+        {
+          name: 'description',
+          content:
+            'Official documentation for RxAngular, the ultimate reactive programming library for Angular.',
+        },
```

Comment:
where did u get that slogan from? :D

Reply:
You don't like It?:D Actually i had no idea what to put there, we can change It